internal/providers/terraform/azure/notification_hubs.go

```diff
@@ -0,0 +1,102 @@
+package azure
+
+import (
+	"fmt"
+
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+	"github.com/tidwall/gjson"
+)
+
+func GetAzureRMNotificationHubsRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "azurerm_notification_hub_namespace",
+		RFunc: NewAzureRMNotificationHubs,
+	}
+}
+
+func NewAzureRMNotificationHubs(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
+	var monthlyAdditionalPushes *decimal.Decimal
+	var startUsageAmt string
+
+	if u != nil && u.Get("monthly_additional_pushes").Type != gjson.Null {
+		monthlyAdditionalPushes = decimalPtr(decimal.NewFromInt(u.Get("monthly_additional_pushes").Int()))
+	}
+
+	sku := "Basic"
+	location := d.Get("location").String()
+
+	if d.Get("sku_name").Type != gjson.Null {
+		sku = d.Get("sku_name").String()
+	}
+	costComponents := make([]*schema.CostComponent, 0)
+
+	costComponents = append(costComponents, notificationHubsCostComponent("Base Charge Per Namespace", location, sku))
```

Comment:
Since each azurerm_notification_hub_namespace is one namespace, should we use "Namespace usage" as the cost component name? That's what we do for "Instance usage" and "Database usage" for other resources.

```suggestion
	costComponents = append(costComponents, notificationHubsCostComponent("Namespace usage", location, sku))
```